site/content/blog/2020-11-01-whats-new-in-svelte-november-2020.md

```diff
@@ -0,0 +1,43 @@
+---
+title: What's new in Svelte: November 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+BLURB SUMMARIZING WHAT'S NEW...
+"What's new in Svelte" series! We'll try to make this a monthly blog post in which you'll find out about new features, bug fixes, and a showcase of Svelte projects from around the community.
+
+## New features
+1. 
+
+**What's going on in Sapper?**
+Any updates here?
+
+
+## Impactful bug fixes
+- 
+
+
+## Coming up
+- Rich Harris demoed the possible future of local Svelte development in a talk titled "Futuristic Web Development" at last month's [Svelte Summit](https://sveltesummit.com/). If you're interested in built-in Server-Side Rendering, modern Hot Module Reloading, or how slots make consistent styling easy across pages, [check out the video](https://www.youtube.com/watch?v=qSfdtmcZ4d0).
```

Comment:
I think this deserves a little more text. I'm not sure though how to do it without turning it into a whole blog post in itself. Thoughts?

Reply:
I think it definitely does deserve more too. I've written [an article](https://mauss.dev/posts/rip-sapper-svelte-is-moving-to-snowpack) about this, I could help sum up and reiterate the important parts or perhaps create another post for this.

Reply:
"Serverless First Framework"

Reply:
Maybe we keep this brief here, we don't want to go into too many details and we want to be very careful about what we put out as officials comms. A dedicated article by those involved with the project makes the most sense.

Reply:
Is my suggested change below already too much or is it okay in your opinion?